.cm/gitstream.cm

```diff
@@ -0,0 +1,77 @@
+# -*- mode: yaml -*-
+# This example configuration for provides basic automations to get started with gitStream.
+# View the gitStream quickstart for more examples: https://docs.gitstream.cm/examples/
```

Comment:
@coderabbitai provide me the exhaustive list of all merged PRs for milestone 1.19.0 that has a bug label.

Reply:
There should be 51 prs

Reply:
Give me the exhaustive list